src/exoscale/tools/project/api/tasks.clj

```diff
@@ -46,7 +46,8 @@
               :for-all [:exoscale.project/modules]}]
 
    :deploy [{:run :exoscale.tools.project.standalone/deploy
-             :unless :exoscale.project/prevent-deploy?
+             :unless :exoscale.project/prevent-deploy? ; legacy
```

Comment:
We don't support both

Reply:
Maybe, maybe not, we have good examples of the opposite in instancepool. I think it's better to make risky/pushy operations explicit in this case. We have also cases where we could infer what to do (uberjar) and we don't, so I suppose it's arguably more consistent.doint it this way.